layers/landuse-labels.yaml

```diff
@@ -0,0 +1,32 @@
+layers:
+    landuse:
+        data: { source: mapzen }
```

Comment:
Remove this `data` line, it's already specified in the layers/landuse.yaml file.